src/styles/reset.scss

```diff
@@ -81,6 +81,9 @@ img::-moz-selection {
 
 ul,
 ol,
+dl,
```

Comment:
Added resets for `<dl>`s and their elements so that default margins don't have to be overridden later.